modules/CmpdReg/src/client/custom/configuration.json

```diff
@@ -1,112 +1,135 @@
-    			{"name": "Parent Stereo Comment", "dataType": "string", "required": false}
+	},
+	"bulkLoadSettings": {
+		"useProjectRoles": false,
+		"dbProperties": [
+				{"name": "CAS Number", "dataType": "string", "required": false},
+				{"name": "Lot Absorbance", "dataType": "numeric", "required": false},
+				{"name": "Lot Alias", "dateType": "string", "required": false},
+				{"name": "Lot Amount Units", "dataType": "string", "required": false},
+				{"name": "Lot Amount", "dataType": "numeric", "required": false},
+				{"name": "Lot Barcode", "dataType": "string", "required": false},
+				{"name": "Lot Boiling Point", "dataType": "numeric", "required": false},
+				{"name": "Lot Chemist", "dataType": "string", "required": true},
+				{"name": "Lot Color", "dataType": "string", "required": false},
+				{"name": "Lot Comments", "dataType": "string", "required": false},
+				{"name": "Lot Corp Name", "dataType": "string", "required": false},
+				{"name": "Lot Is Virtual", "dataType": "boolean", "required": false},
+				{"name": "Lot Lambda", "dataType": "numeric", "required": false},
+				{"name": "Lot Melting Point", "dataType": "numeric", "required": false},
+				{"name": "Lot Notebook Page", "dataType": "string", "required": false},
+				{"name": "Lot Number", "dataType": "numeric", "required": false},
+				{"name": "Lot Observed Mass #1", "dataType": "numeric", "required": false},
+				{"name": "Lot Observed Mass #2", "dataType": "numeric", "required": false},
+				{"name": "Lot Percent ee", "dataType": "numeric", "required": false},
+				{"name": "Lot Physical State", "dataType": "string", "required": false},
+				{"name": "Lot Purity Measured By", "dataType": "string", "required": false},
+				{"name": "Lot Purity Operator", "dataType": "string", "required": false},
+				{"name": "Lot Purity", "dataType": "numeric", "required": false},
+				{"name": "Lot Retain Location", "dataType": "string", "required": false},
+				{"name": "Lot Retain Units", "dataType": "string", "required": false},
+				{"name": "Lot Retain", "dataType": "numeric", "required": false},
+				{"name": "Lot Salt Abbrev", "dataType": "string", "required": false},
+				{"name": "Lot Salt Equivalents", "dataType": "string", "required": false},
+				{"name": "Lot Solution Amount Units", "dataType": "string", "required": false},
+				{"name": "Lot Solution Amount", "dataType": "numeric", "required": false},
+				{"name": "Lot Stock Location", "dataType": "string", "required": false},
+				{"name": "Lot Stock Solvent", "dataType": "string", "required": false},
+				{"name": "Lot Storage Location", "dataType": "string", "required": false},				
+				{"name": "Lot Supplier ID", "dataType": "string", "required": false},
+				{"name": "Lot Supplier Lot", "dataType": "string", "required": false},
+				{"name": "Lot Supplier", "dataType": "string", "required": false},
+				{"name": "Lot Synthesis Date", "dataType": "date", "required": false},
```

Comment:
@bffrost don't we almost always have the date marked as required? Seems like it wouldn't be a bad default setting.

Reply:
We have had requests in the past to have the required set be only the set actually required by the database, so the most minimal set. This is why Lot Synthesis Date got changed (a couple versions ago) from required to not in the default bulk loader configs.

Reply:
Ah okay cool. I was thinking about it in the context of the single reg gui anyway. Makes sense!